content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -2,40 +2,122 @@
 Cycle counts
 ============
 
-In most companies, the stock is only counted once a year. That's why by default, after making an
-inventory adjustment in Odoo, the scheduled date for the next count is set on the 31st of
-December. However, for some businesses it's crucial to have an accurate inventory count at all
-times.
+For most companies, warehouse stock only needs to be counted once a year. This is why, by default,
+after making an *inventory adjustment* in Odoo, the scheduled date for the next inventory count is
+set for the 31st of December of the current year.
 
-The goal of cycle counts is to keep critical stock levels accurate by counting more often at key
-locations.
+However, for some businesses, it's crucial to have an accurate inventory count at all times. These
+companies use *cycle counts* to keep critical stock levels accurate. Cycle counting is a method by
+which companies count their inventory more often in certain *locations*, to ensure that their
+physical inventory counts match their inventory records in Odoo.
 
-Configuration
-=============
+Activate storage locations
+==========================
 
-In Odoo, cycle counts are location-based. The frequency of the counts is defined by the storage
-location. To activate storage locations, go to :menuselection:`Inventory --> Configuration -->
-Settings --> Warehouse` and activate the :guilabel:`Storage Locations` setting. Next, click
-:guilabel:`Save` to apply the setting.
+In Odoo, cycle counts are location-based. Therefore, the *storage locations* feature needs to be
+enabled before performing a cycle count.
```

Comment:
```suggestion
In Odoo, cycle counts are location-based. Therefore, the :guilabel:`Storage Locations` feature
needs to be enabled before performing a cycle count.
```
added a guilabel here because Storage Locations is the name of a feature.